buildkite/resource_organization_settings_test.go

```diff
@@ -86,24 +86,6 @@ func TestAccOrganizationSettings_import(t *testing.T) {
 	})
 }
 
-func TestAccOrganizationSettings_disappears(t *testing.T) {
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
-		ProtoV6ProviderFactories: protoV6ProviderFactories(),
-		CheckDestroy:             testCheckOrganizationSettingsResourceRemoved,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccOrganizationSettingsConfigBasic([]string{"0.0.0.0/0", "1.1.1.1/32", "1.0.0.1/32"}),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					// Confirm that the allowed IP addresses are set correctly in Buildkite's system
-					testAccCheckResourceDisappears(Provider("testing"), resourceOrganizationSettings(), "buildkite_organization_settings.let_them_in"),
```

Comment:
I've removed this whole test because I've refactored this function. This resource is going away soon anyway so we'll be deleting everything. I figure that removing this a little early is probably fine as well

Reply:
I think it'll be a similar story for all the SDKv2 implemented resources - but thats fair